nais/nais.yaml

```diff
@@ -39,3 +39,9 @@ spec:
     requests:
       cpu: "1"
       memory: 2560Mi
+  kafka:
+    pool: nav-dev
+  accessPolicy:
+    outbound:
+      external:
+        - host: kafka-schema-registry.nais-q.adeo.no
```

Comment:
Burde vi ha denne som en variabel? Eller er det samme adresse i dev og prod?

Reply:
Nei, det er nok ikke samme adresse. Så ja, kan legge den i en variabel. Vi tenker vel egentlig å gå bort fra dette når vi går til GCP, og heller ha to ulike nais.yaml-filer (dev og prod).